target/linux/ath79/image/generic.mk

```diff
@@ -1540,6 +1540,13 @@ define Device/engenius_ews660ap
 endef
 TARGET_DEVICES += engenius_ews660ap
 
+define Device/engenius_ens1750
+  $(Device/engenius_ews660ap)
```

Comment:
It's better to create a common definition for ews660ap and ens1750 instead of calling `Device/engenius_ews660ap` directly. There are a lot of examples in this file.

Reply:
Created `Device/engenius_dual_ap` [here](https://github.com/openwrt/openwrt/blob/372f66a2b5c68b469c31081af2686ff036309aaf/target/linux/ath79/image/generic.mk#L1531). Is that what you meant?

Reply:
Yes. LGTM.